### DISCUSSION TITLE
New n64 Flashrom to add please

### DISCUSSION THREAD

**Discussion by fiatbravodriver** [sanni/cartreader#188] [Coding Corner]
Hi, 

I have found a flashrom to add please:

![IMG_2610 (1)](https://user-images.githubusercontent.com/43493640/125162069-e2f80a00-e17d-11eb-9c69-7215c0f21158.jpg)
![IMG_2598](https://user-images.githubusercontent.com/43493640/125162076-edb29f00-e17d-11eb-9b3f-e02a06507485.jpg)

https://www.datasheets360.com/part/detail/s29gl512n11tfi02/4736602268136894754/

Here is a link to what I believe is the correct data sheet.
Happy to test as always. 

Kind Regards
J

**Comment by fiatbravodriver** [comment]
Bump

**Comment by sanni** [comment]
Have you tried the manual config?
According to the datasheet it has 128 Kbyte sectors and a 32 byte write buffer.

**Comment by fiatbravodriver** [comment]
Hi, for some very strange reason I have just tried again and it flashed as normal. I am very confused but glad it is not a wasted cart and thanks for taking the time to reply. 
J


-----Original Message-----

**Reply by Dakkaron** [reply]
I have a chip of the same series on a gb repro cart. There the ID command always returned different ID values...
